jujugui/static/gui/src/app/components/inspector/header/header.js

```diff
@@ -65,6 +119,8 @@ class InspectorHeader extends React.Component {
 InspectorHeader.propTypes = {
   activeComponent: PropTypes.string,
   backCallback: PropTypes.func.isRequired,
+  changeState: PropTypes.func, // Not required as charm may not include gettingstarted.md
```

Comment:
What do you think about:
- having changeState required;
- having hasGettingStarted as a boolean property, so that it's very explicit and not relying on the presence of a arguably not strictly related property;
- changing entityid to entityURL and pass in the URL object rather than a string, so that we don't have to do the try/except parsing dance above?